nightwatch.conf.js

```diff
@@ -49,5 +49,12 @@ module.exports = {
         },
       },
     },
+    'firefox-headless': {
+      desiredCapabilities: {
+        chromeOptions: {
+          args: ['--headless', '--disable-gpu', 'window-size=1600,1200'],
+        },
```

Comment:
Can you try adding this instead? 
```
'moz:firefoxOptions': {
   args: ['-headless']
 }
```

Obviously `chromeOptions` shouldn't have any effect here..